### DISCUSSION TITLE
What does the `output.asyncChunks` option actually do?

### DISCUSSION THREAD

**Discussion by iamgreenintro** [webpack/webpack#17935] [Q&A]
The option I'm talking about (and notice it's lacking documentation):
https://webpack.js.org/configuration/output/#outputasyncchunks

I am having a hard time figuring out what exactly the option does and how it works.
I'm aware of what asynchronous means and that it is most likely useful in, for example, a web app with multiple routes/pages that each have their own entry point. What about an application that has a single entry point? Does asyncChunks do anything in that case?

Are the asyncChunks refering to the entry points defined in `entry { ... }`? Please elaborate it for me as any extra information is useful!

**Comment by alexander-akait** [comment]
If `true` - it means your application will have async chunks (it based on how your wrote code or configure using `webpack.config.js`, you can have small application without on demand chunks), if you set to `false`, webpack generate bundle without dynamic loaded chunks (on demand chunks, i.e. all dynamically loaded modules will be loaded like initial), think about it as dynamic `import(...)`, I recommend do not disable it and do not touch if you really don't understand why you need to disable it.
